crates/provider/src/new.rs

```diff
@@ -348,9 +360,9 @@ pub trait Provider<N: Network, T: Transport + Clone = BoxTransport>: Send + Sync
     async fn send_transaction(
         &self,
         tx: N::TransactionRequest,
-    ) -> TransportResult<PendingTransactionConfig> {
+    ) -> TransportResult<PendingTransactionBuilder<'_, N, T>> {
```

Comment:
can we make pending tx not borrowing?
this makes it impossible to move the pending tx somewhere else

Reply:
ah I may have misunderstood this, this is only the first step, this can be converted into an owned by calling register etc..

Reply:
yep this is just a local builder